worlds/spire/Options.py

```diff
@@ -37,5 +59,6 @@ class HeartRun(Toggle):
 spire_options: typing.Dict[str, type(Option)] = {
     "character": Character,
     "ascension": Ascension,
-    "heart_run": HeartRun
+    "final_act": FinalAct,
```

Comment:
yes, my mod will accept the old name of this as well for backwards compatibility.